packages/table-core/src/features/Pagination.ts

```diff
@@ -130,7 +130,7 @@ export const Pagination: TableFeature = {
               ? Number.MAX_SAFE_INTEGER
               : table.options.pageCount - 1
 
-          pageIndex = Math.min(Math.max(0, pageIndex), maxPageIndex)
```

Comment:
These only differ in the case where maxPageIndex is < 0, which can only happen when pageCount <= 0